modules/nf-core/phispy/main.nf

```diff
@@ -0,0 +1,37 @@
+process PHISPY {
+    tag "$meta.id"
+    label 'process_medium'
+
+    conda "bioconda::phispy=4.2.21"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/phispy:4.2.21--py310h30d9df9_1':
+        'biocontainers/phispy:4.2.21--py310h30d9df9_1' }"
+
+    input:
+    tuple val(meta), path(gbk)
+
+    output:
+    tuple val(meta), path("${prefix}/")                        , emit: results
+    tuple val(meta), path("${prefix}/prophage_coordinates.tsv"), emit: coordinates
+    tuple val(meta), path("${prefix}/*.gb*")                   , emit: gbk
+    path "versions.yml"                                        , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    prefix = task.ext.prefix ?: "${meta.id}"
+    """
+    PhiSpy.py \\
+        $args \\
+        --threads $task.cpus \\
+        -o $prefix \\
+        $gbk
+
```

Comment:
Can you move all files in the output folder and rename them to contain the prefix? This makes sure that the output has a unique name

Reply:
Sure! By that you mean add a 
```
mv $prefix/prophage_coordinates.tsv {prefix}/${prefix}_prophage_coordinates.tsv 
```
something like that?

Reply:
Something like that yes, maybe also drop the `_prohpage_coordinates` so the file name is completely customizable